### PR TITLE
feat: IC external integrations

### PR DESCRIPTION
**Pull request by Darkheir**
Adds a page about external integrations for the intelligence center.

@otetard Please wait for Gaël's approval before merging. Of course don't hesitate to review it ;-)

### CODE REVIEW

File: docs/intelligence_center/integrations.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Intelligence Center external integrations
+
+The intelligence center data can be consummed using several third party integrations.
```
Comment (otetard) [inline docs/intelligence_center/integrations.md:3]:
```suggestion
Intelligence Center data can be consumed using several third party integrations.
```

File: docs/intelligence_center/integrations.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+-   Source Format: MISP Feed
+-   Headers: `Authorization: Bearer <APIKEY>`
+-   Enabled: `True`
+
+You then need to make sure you have a scheduled task in place to regularly fetch the feed's content.
+
+## OpenCTI connector
+
+An [OpenCTI](https://www.opencti.io) connector is available to consumme a feed.
+All the instruction to run it are available at the connector github repository: https://github.com/SEKOIA-IO/opencti-connector.
```
Comment (otetard) [inline docs/intelligence_center/integrations.md:24]:
```suggestion
All the instruction to run it are available at the connector GitHub repository: https://github.com/SEKOIA-IO/opencti-connector.
```

File: docs/intelligence_center/integrations.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Cortex Analyser
+
+SEKOIA is also providing a [cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [THEHIVE](https://thehive-project.org/) ecosystem.
```
Comment (otetard) [inline docs/intelligence_center/integrations.md:28]:
```suggestion
SEKOIA is also providing a [Cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [TheHive](https://thehive-project.org/) ecosystem.
```

File: docs/intelligence_center/integrations.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+The default feed is available as a MISP feed.
+
+It can be added to an existing MISP instance by following [MISP's documentation](https://www.circl.lu/doc/misp/managing-feeds/).
+
+The following field values are required for the feed to work properly:
+
+-   Input Source: Network
+-   URL: `https://api.sekoia.io/api/v2/inthreat/misp`
+-   Source Format: MISP Feed
+-   Headers: `Authorization: Bearer <APIKEY>`
```
Comment (otetard) [inline docs/intelligence_center/integrations.md:16]: Shouldn’t we also explain how to manage APIKeys? BTW, it seems that we don’t have such a page currently.

File: docs/intelligence_center/integrations.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+## OpenCTI connector
+
+An [OpenCTI](https://www.opencti.io) connector is available to consumme a feed.
+All the instruction to run it are available at the connector github repository: https://github.com/SEKOIA-IO/opencti-connector.
+
+## Cortex Analyser
+
+SEKOIA is also providing a [cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [THEHIVE](https://thehive-project.org/) ecosystem.
+To setup the analyzer please follow the following guide: https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md.
```
Comment (otetard) [inline docs/intelligence_center/integrations.md:29]:
```suggestion
To setup the analyzer please [follow the following guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```

**Review by otetard — APPROVED** [review]

File: docs/intelligence_center/integrations.md (hunk 4 above)
Comment (Darkheir) [reply docs/intelligence_center/integrations.md:16]: Yes I agree that it would be useful. We should create a page and add a link to it in this section.

Comment (celinequivy) [reply docs/intelligence_center/integrations.md:16]: I planned to add pages concerning all the community/user management part soon, but if someone wants to do it, fell free !

File: docs/intelligence_center/integrations.md (hunk 5 above)
Comment (celinequivy) [reply docs/intelligence_center/integrations.md]: To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).

File: docs/intelligence_center/integrations.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+## OpenCTI connector
+
+An [OpenCTI](https://www.opencti.io) connector is available to consumme a feed.
+All the instruction to run it are available at the connector GitHub repository: https://github.com/SEKOIA-IO/opencti-connector.
+
+## Cortex Analyser
+
+SEKOIA is also providing a [Cortex analyzer](https://github.com/TheHive-Project/Cortex-Analyzers/tree/master/analyzers/SEKOIAIntelligenceCenter) to enrich data in [TheHive](https://thehive-project.org/) ecosystem.
+To setup the analyzer please [follow the following guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```
Comment (Darkheir) [inline docs/intelligence_center/integrations.md:29]:
```suggestion
To setup the analyzer please [follow this guide](https://github.com/TheHive-Project/CortexDocs/blob/master/analyzer_requirements.md).
```

**Review by celinequivy — APPROVED** [review]

**Review by gaelmuller — APPROVED** [review]

**Pull request merged**